pkg/odo/cli/dev/dev.go

```diff
@@ -37,8 +44,10 @@ type DevOptions struct {
 	clientset *clientset.Clientset
 
 	// Variables
-	ignorePaths []string
-	out         io.Writer
+	ignorePaths       []string
+	out               io.Writer
+	errOut            io.Writer
+	initialDevfileObj parser.DevfileObj // it's called "initial" because it has to be set only once when running odo dev for the first time
```

Comment:
You could explain the purpose of this initial devfile. As I understand, to compare it with the future updated devfile

Reply:
Yes, I could add that as well. But I do want to keep what I have already put there as well, because at some point we might start using it for some other thing instead.